tac/project-updates/2024/2024-Q4-Hyperledger-Web3j.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: 2024 Q4 Hyperledger Web3j
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+
+- The project is in good health.
+- We did 11 releases for all the Web3j Repos fixing several bugs, issues, vulnerabilities, licenses and adding new features.
+- Added steps and guidelines on how to become a Web3j maintainer.
+- Mentee Shashank is actively working on Web3j-android project through Hyperledger Mentorship Program.
+- Regularly having fortnightly contributors call.
+- Added Web3j [Roadmap](https://lf-hyperledger.atlassian.net/wiki/spaces/WEB3J/pages/23101932/Roadmap+2024+-+2025).
+
+# Questions/Issues for the TOC
+
+None at the moment.
+
+# Releases
+
+We did release several strong releases for all Web3j repos.
+
+- Web3j Core
+  - v4.12.1 (Aug 14, 2024)
+  - v4.12.2 (Sept 18, 2024)
+  
+- Web3j Gradle Plugin
+  - v4.12.1 (Aug 16, 2024)
+  - v4.12.2 (Sept 19, 2024)
+  
+- Web3j Maven Plugin
+  - v4.12.1 (Aug 16, 2024)
+  
+- Web3j Unit
+  - v4.12.1 (Aug 15, 2024)
+  - v4.12.2 (Sept 18, 2024)
+
+- Web3j EVM
+  - v4.12.1 (Aug 15, 2024)
+  - v4.12.2 (Sept 18, 2024)
+
+- Web3j Solidity Gradle Plugin
+    - v0.5.1 (Aug 16, 2024)
+
+- Web3j CLI
+    - v1.6.1 (Aug 17, 2024)
+
+# Overall Activity in the Past Quarter
+
+- 33 commits and 10 PRs merged in Web3j core.
+- 50+ commits and 18 PRs merged across all other Web3j repositories.
+- Fixed bugs, issues, and vulnerabilities, and updated dependencies.
+- Fixed licenses issue with some files in Web3j-repos.
+- Added support for Linea ENS.
+- Hyperledger Mentorship Program - 
+  - Mentee actively contributing to new Web3j-android [branch](https://github.com/shashankiitbhu/web3j/tree/web3j-android).
+  - Converted Web3j core to an android project and added android based configs. Resolving build issues.
+  - Tested Web3j functionalities in an android project.
+- Regularly having fortnightly contributors call. A few people joined the call, but it is not very successful yet.
+- The Discord channel was active with questions and announcements of new releases.
+
+# Current Plans
+
+- Fixing Bugs and issues in Web3j.
+- Getting web3j-android fixed for newer Android versions.
+- Adding support for account abstraction in Web3j.
+- Fix Web3j open-api project.
+- Organize Web3j Summit in early December.
+
+# Maintainer Diversity
+
+- No new maintainers were added this quarter, but several contributors remained active and showed interest towards maintainership.
```

Comment:
Ideally you should summarize the current maintainer diversity rather than just mentioning any changes.

Reply:
^^ +1